Makefile

```diff
@@ -53,21 +54,38 @@ $(VIRTUALENV)/bin/python3:
 	python3 -m venv $(VIRTUALENV)
 	$(VIRTUALENV)/bin/pip install --upgrade pip
 
+.PHONY: venv
+venv: $(VIRTUALENV)/bin/python3
+
+.PHONY: release
+release:
+	make venv VIRTUALENV=$(RELEASE_TEST_VENV)
+	$(RELEASE_TEST_VENV)/bin/pip3 install wheel
+	$(RELEASE_TEST_VENV)/bin/python3 setup.py sdist bdist_wheel
+	$(RELEASE_TEST_VENV)/bin/pip install dist/*.whl
+	$(RELEASE_TEST_VENV)/bin/pip3 install -r requirements-dev.txt
+
+.PHONY: release-test
+release-test: release
+	. $(RELEASE_TEST_VENV)/bin/activate && pytest --docker-image=ubuntu:bionic
+
 # Install development dependencies (for testing) in virtualenv
 .PHONY: dev
-dev: $(VIRTUALENV)/bin/python3
+dev: venv
 	$(VIRTUALENV)/bin/pip3 install -e '.[dev]'
 
 # Clean directory and delete virtualenv
 .PHONY: clean
 clean:
-	$(VIRTUALENV)/bin/python3 setup.py clean --all
+	-$(VIRTUALENV)/bin/python3 setup.py clean --all
+	-$(RELEASE_TEST_VENV)/bin/python3 setup.py clean --all
```

Comment:
What are the minus signes for at before the executable path?

Reply:
That means do not care about the return value of the command.
This can be an equivalent expression `$(VIRTUALENV)/bin/python3 setup.py clean --all || true`.
I prefer the minus sign, because it is a standard in makefiles and systemd units.